### PR TITLE
fix unixy bug

### PR DESCRIPTION
**Pull request by amyburden**

### CODE REVIEW

**Comment by 791136190** [comment]
这样修改有验证过结果吗，还有就是需要使用sum？

**Comment by amyburden** [comment]
那你等我训练训练再看，不用sum也可以，就是像你一样调整权重，这块可能还有点问题，那你等我复现出来再说吧
idx = torch.argsort(position).detach()  # 返回的索引是0开始的 上面的方式loss会略大0.000x级别
p = position.shape[0]
idx_f = torch.arange(p).float()
uni_l2 = torch.mean(torch.pow(position[idx] - (idx_f / p), 2))
